tools/devtool

```diff
@@ -562,7 +562,14 @@ ensure_ci_artifacts() {
 
     # Fetch all the artifacts so they are local
     say "Fetching CI artifacts from S3"
-    S3_URL=s3://spec.ccfc.min/firecracker-ci/v1.11/$(uname -m)
+
+    FC_VERSION=$(run_devctr \
```

Comment:
Can we re-use `cmd_sh` here?

Reply:
we cannot because it calls ensure_ci_artifacts :(

Reply:
Can we decouple it so that every existing call site of `cmd_sh`, we call `cmd_download_artifacts` first (if applicable)? I think in most cases it was unneeded but we had no way of controlling it.